### PR TITLE
Canvas modules

### PR DESCRIPTION
**Pull request by frankfusco**
Adds endpoints to track Canvas module and course completion:

- `canvasRouter.get("/courses/:courseId/modules/completion", getCourseModuleCompletion);`
- `canvasRouter.get("/courses/:courseId/modules/:moduleId", getModule);`
- `canvasRouter.get("/courses/:courseId/modules/:moduleId/completion", getModuleCompletion);`
- `canvasRouter.get("/courses/:courseId/completed", getCourseCompleted);`
- `canvasRouter.get("/courses/completed", getAllRequiredCoursesCompleted);`

"Completion" endpoints return completion information, a subset of the info returned from the Canvas API with the:

- id
- name
- state (e.g. "completed")
- completed_at (date)

for each module.

"Completed" endpoints return booleans.

Both types require appending `lambdaId=____` as a query param.

This should help Release Managers check off final sprint milestones + objectives related to module/course completion.

### CODE REVIEW

**Comment by tinomen** [comment]
Are these meant to be used on a per student basis?

**Comment by frankfusco** [comment]
> Are these meant to be used on a per student basis?

Yup

**Review by tinomen — APPROVED** [review]
Looks fine. 

I'm not sure I understand the strategy this will fulfill.

File: docs/teambuilding.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Teambuilding API Contract
```
[the remark this answers is not in the document]
Comment (frankfusco) [reply docs/teambuilding.md:1]: Yeah we got a little off the rails there haha

**Comment by frankfusco** [comment]
> Looks fine.
> 
> I'm not sure I understand the strategy this will fulfill.

Short-term, just a way to let RMs pull this info in Postman per student and go through to check things off, then finally verify they actually completed a course for the graduation objectives, then confirm they've actually completed all the courses before signing them off for graduation.

**Pull request merged**